docs/modules/vector-search/pages/vector-collections.adoc

```diff
@@ -0,0 +1,498 @@
+= Vector Collection
+
+== Intro
+
+The primary object for interacting with the database is a Vector Collection. This collection holds information about the vectors and metadata (value).
+
+A collection can consist of one or more indexes, all sharing a single storage metadata. Each index represents a distinct vector space. Vectors from different indexes are stored independently, and indexes can have different configurations.
+
+NOTE: At this point, each metadata object must correspond to exactly one vector in each index.
+
+NOTE: When working with multiple indexes in your collections, it's necessary to specify the index to search in your queries..
+
+== Configuration
+
+Collection configuration can be set dynamically during vector collection creation or statically during cluster configuration. Below is a table listing all available options:
+
+.Collection configuration options
+[cols="1,2,1,1",options="header"]
+|===
+|Option|Description|Required|Default
+
+|name
+|The name of the vector collection.
+|Y
+|null
+
+|indexes
+|Information about indexes configuration
+|Y
+|null
+|===
+
+.Index configuration options
+[cols="1,2,1,1",options="header"]
+|===
+|Option|Description|Required|Default
+
+|name
+|The name of the vector index.
+|`N` - for single-index vector collections. `Y` - for multi-index collection
+|null
+
+|dimension
+|Vectors dimension
+|Y
+|null
+
+|metric
+|Distance metric used to calculate the distance between two vectors.
+See table Available distance metrics.
```

Comment:
I know it's right next to this table, but perhaps best to replace with a link to `Available distance metrics`

Reply:
6b265566fb67a94493a066d7015575e597f5ece5